modules/ROOT/pages/_partials/secrets-manager.adoc

```diff
@@ -0,0 +1,713 @@
+
+This command accepts the default flag `--help`.
+
+NOTE: The output does not include any sensitive or secret data.
+
+
+[[secret-TLS-context-list]]
+== secrets-mgr:tls-context:mule:list
+
+----
+> secrets-mgr:tls-context:mule:list [flags]
+----
+
+Lists all Mule TLS Context secrets in a secret group specified in `--group-id`
+
+This command accepts the default flag `--help`.
+
+[[secret-TLS-context-modify]]
+== secrets-mgr:tls-context:mule:modify
+
+----
+> secrets-mgr:TLS-context:mule:modify  [flags] 
+----
+
+Modifies the name or expiration date for a Mule TLS Context secret specified in `--id` from the group you pass in`--groupid`. 
+
+Besides the default flag `--help`, this command also accepts:
+
+[%header,cols="30a,40a,30a"]
+|===
+|Flag | Description |  Example
+|`--name` | New name for the truststore secret | `--name TestTruststoreSecret`
+|`--expiration-date` | New expiration date for the truststore secret | `--expiration-date 01/01/2025`
+
+|===
+
+[[secrets-TLS-context-replace]]
+== secrets-mgr:tls-context:mule:replace
+
+----
+> secrets-mgr:tls:context:mule:replace  [flags] 
+----
+
+Replaces an existing Mule TLS Context secret you pass in `--id`, from the secret group you pass in `--group-id`, using the type you pass in `--type`. 
+
+Besides the default flag `--help`, this command also accepts:
+
+[%header,cols="30a,40a,30a"]
+|===
+|Flag | Description |  Example
+|`--group-id`| Mandatory. Secret group ID. | `--group-id 030493`
+|`--name`|Mandatory. Name for your secret. | `--name TestSecret`
+|`--tls-version` |TLS Version +
+Default: TLSv1.2. | `--tls-version TLSv1.1`
+|`--cipher` |Cipher for the specified TLS version. | `--cipher      TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256`
+|`--expiration-date`| Expiration date for the secret| `--expiration-date 01/01/2025`
+|`--insecure`|Disable certificate validation. | `--insecure`
+|`--keystore-id`|A valid JKS, JCEKS or PKCS12 keystore ID in the secret group, which is used as keystore for the TLS context. | `--keystore-id 00003443`
+|`--truststore-id`|A valid JKS, JCEKS or PKCS12 truststore ID in the secret group, which is used as truststore for the TLS context. | `--truststore-id 001111032`
+
+|===
+
+[[secret-TLS-flex-create]]
+== secrets-mgr:tls-context:flex-gateway:create
+
+----
+> secrets-mgr:tls-context:flex-gateway:create [flags] 
+----
+
+Creates a new Flex Gateway TLS Context secret in the secret group specified in `--group-id`, and using the name specified in `--name`.
+
+Besides the default flag `--help`, this command also accepts:
+
+[%header,cols="30a,40a,30a"]
+|===
+|Flag | Description |  Example
+|`--group-id`| Mandatory. Secret group ID. | `--group-id 030493`
+|`--name`|Mandatory. Name for your secret. | `--name TestSecret`
+|`--max-tls-version` |Maximum TLS Version +
+Default: TLSv1.3. | `--tls-version TLSv1.1`
+|`--min-tls-version` |Minimum TLS Version +
+Default: TLSv1.1. | `--tls-version TLSv1.2`
```

Comment:
In the example, this should be `--min-tls-version` and not `--tls-version`. Also, use TLSv1.2 instead of TLSv1.1